ui/pages/confirmations/components/confirm/info/personal-sign/personal-sign.tsx

```diff
@@ -33,9 +33,25 @@ const PersonalSignInfo: React.FC = () => {
   }
 
   const { from } = currentConfirmation.msgParams;
+  const siweSignatureRequest = isSIWESignatureRequest(currentConfirmation);
```

Comment:
[nit] siweSignatureRequest → isSiweSignatureRequest
prepend 'is' as function returns a boolean value